src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(s -> Arrays.stream(s.split(",")))
+                .map(Integer::parseInt)
+                .filter(n -> n % 2 == 0)
+                .min(Integer::compareTo)
```

Comment:
similar approach here
```suggestion
                .mapToInt(Integer::parseInt)
                .filter(n -> n % 2 == 0)
                .min()
```